src/index.html

```diff
@@ -3,10 +3,452 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page">
+    <header class="header">
+      <div class="icon">
+        <a href="#head" class="icon__link"><img src="./images/logo.png" alt="logo"></a>
+      </div>
+      <nav class="nav nav--position">
+        <ul class="nav__list">
+          <li class="nav__item"><a href="#apple " class="is-active">apple</a></li>
+          <li class="nav__item"><a href="#samsung">samsung</a></li>
+          <li class="nav__item"><a href="#smartphones">smartphones</a></li>
+          <li class="nav__item" data-qa="nav-hover"><a href="#laptops&computers">laptops & computers</a></li>
+          <li class="nav__item"><a href="#gadgets">gadgets</a></li>
+          <li class="nav__item"><a href="#tablets">tablets</a></li>
+          <li class="nav__item"><a href="#photo">photo</a></li>
+          <li class="nav__item"><a href="#video">video</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main class="content content--position">
+      <section class="content__row">
+        <div class="card content__card" data-qa="card">
```

Comment:
Consider using a semantic tag for the card to improve the accessibility of your code